### PR TITLE
Windows: Appveyor Config + Release Artifacts

### PR DESCRIPTION
**Pull request by bryphe**

### CODE REVIEW

**Comment by bryphe** [comment]
I'm also working on, as part of this PR, updating our `platform-release` script to accomodate Windows, as well as our release script.

File: appveyor.yml [hunk 1]
```diff
@@ -21,4 +21,16 @@ artifacts:
       name: Windows Build
       type: File
 
+deploy:
+    - provider: GitHub
+      repository: andreypopp/esy-solve-cudf
+      auth_token:
+          secure: TODO
```
Comment (bryphe) [inline appveyor.yml]: @andreypopp - FYI - we'll need your __encrypted__ token here:
```
GitHub authentication token (auth_token) - OAuth token used for authentication against GitHub API. You can generate Personal API access token at https://github.com/settings/tokens. Minimal token scope is repo or public_repo to release on private or public repositories respectively. Be sure to encrypt your token using the Account → Encrypt data tool.
```

Once you get your Github token - you can encrypt it with this tool on AppVeyor: https://ci.appveyor.com/tools/encrypt

Thanks & let me know if you need any help!

**Comment by bryphe** [comment]
This should be good to go now, @andreypopp !

I checked the release artifact here, and looks good: https://ci.appveyor.com/project/andreypopp/esy-solve-cudf/build/1.0.7/artifacts

In addition, I tried a mock `make release` from my fork, and it pulled in the windows binaries as expected.

The only thing we need is the secure token - we can add that now or as a separate change (whatever you prefer).

**Comment by andreypopp** [comment]
Ok, added token, thank you, @bryphe!

Seeing esy running on windows is so cool! Going to cut a new release of esy-solve-cudf now with win binaries.

**Pull request merged**